frontend/Components/ListingCard/ListingCard.module.css

```diff
@@ -0,0 +1,41 @@
+.cardContainer {
```

Comment:
Ya que usamos CSS modules, esto se puede llamar:

```suggestion
.container {
```